src/connectors/cloudflare/index.ts

```diff
@@ -65,7 +64,11 @@ export class CloudflareService {
     return key
   }
 
-  baseUploadFile = async (folder: GQLAssetType, upload: any, uuid: string) => {
+  public baseUploadFile = async (
+    folder: GQLAssetType,
+    upload: any,
```

Comment:
not a big fan whenever it comes to `any`. We can explicitly type it with just the types we will be using here:
`const { createReadStream, mimetype, filename } = upload`